src/main/java/org/takes/facets/fork/FkHitRefresh.java

```diff
@@ -172,7 +164,16 @@ private boolean expired() {
      * @throws IOException If fails
      */
     private void touch() throws IOException {
```

Comment:
@longtimeago Actually it is still needed as you can see here https://github.com/yegor256/takes/blob/5a5968552bb9b0de1ad4b94c1a07cc6450b6405c/src/main/java/org/takes/facets/fork/FkHitRefresh.java#L142